qemu/qemu.go

```diff
@@ -58,7 +59,7 @@ type Device interface {
 // DeviceDriver is the device driver string.
 type DeviceDriver string
 
-const (
+var (
```

Comment:
**Please** don't remove the `const` here :smile:

Reply:
I am setting the value of these variables at runtime. So making them const is not really possible. 

However, since you and others have raised concerns with this and the arch specific if-else, let me just refactor this into a separate file new file.